packages/planes-ds/src/lib/components/atoms/input/integerInput.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react"
+import { z } from "zod"
+
+//import { cn } from "@/lib/utils"
+import { cn } from "../../../utils"
+import { useState, useEffect } from 'react';
+
+export interface InputProps
+    extends React.InputHTMLAttributes<HTMLInputElement> {
+        isRequired: boolean;
+    }
+
+const IntegerInput = React.forwardRef<HTMLInputElement, InputProps>(
+    ({ className, type, isRequired, value, ...props }, ref ) => {
+        //Zod validation with regular expressions
+        const schema = z.string().regex(/^\d*$/, 'Must be an Integer');
+        //^\d*$ meaning 
+        //"^" = start of string
+        //"\d" = is any number digit (0-9)
+        //"*" = 0 or more
+        //"$" = Final of string
+
+        //Constant using for show zod error
+        const [error, setError] = useState(null);
+        
+        const handleInputChange = (e : any) => {
+            const newValue = e.target.value;    //new value = value from input
+            //validation of value
+            try{
+                schema.parse(newValue); //zod validation
```

Comment:
Do we need to create a new value? Or we can just validate inline?
```suggestion
                schema.parse(e.target.value); //zod validation
```

Reply:
We can use it inline, I check and it works! 
Is not necesesary create a new variable